.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,139 @@
+name: Maintenance
+on:
+    workflow_dispatch:
+        inputs:
+            signing-key-id:
+                required: true
+                type: string
+                default: C8B77BCBE16CD2B94B43F9C8757397B82D4ED7B0
+
+jobs:
+    create-branch-prefix:
```

Comment:
Shouldn't there be outputs here?

Reply:
Apparently when you do a reusable workflow like this you CANNOT specify the outputs except in the workflow yml.